app/code/AstoundDRudenko/Badge/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <section id="catalog">
+            <group id="product_badges" translate="label" type="text" sortOrder="345" showInDefault="1" showInWebsite="1" showInStore="0">
```

Comment:
please implement acl resources for product_badges group

Reply:
In magento 2 we can't configure an acl resource directly in system.xml file. Please see allowed attributes
for section: `app/vendor/magento/module-config/etc/system_file.xsd:103`
for group: `app/vendor/magento/module-config/etc/system_file.xsd:138`
But we can create an acl resource and configure permissions via events or plugins in future.